core/drand_beacon_public.go

```diff
@@ -189,7 +189,7 @@ func (bp *BeaconProcess) SignalDKGParticipant(ctx context.Context, p *drand.Sign
 	bp.state.Lock()
 	if bp.manager == nil {
 		bp.state.Unlock()
-		return nil, errors.New("no manager")
+		return nil, errors.New("no DKG in progress; make sure you are using the correct --id flag for the chain")
```

Comment:
Same comment as in your other PR actually: it would be best to not mention flags within the library part of drand and instead catch this error on the client side and issue proper warning there, no?

Reply:
Right now we can't get a `Beacon` from inside the cmd.  
We could expose the `manager` or `isDoingDKG()` or something via the control client potentially

Reply:
How about just making it more generic then and not mention the flags?
Maybe:
``` 
fmt.Errorf("no DKG in progress for this Beacon ID")
```